src/components/HomepageFeatures/styles.module.css

```diff
@@ -9,3 +9,9 @@
   height: 200px;
   width: 200px;
 }
+
+.wideLogoImage {
+  width: 100%;
+  height: auto;
+  max-width: 600px;
```

Comment:
Why is the max-width set to 600px here?

Reply:
The logo in the top left corner of the page is approximately 300px. Making it about twice this size would look more comfortable.